lib/app.ts

```diff
@@ -5,6 +5,11 @@ const isObject = o => typeof o === 'object' && o !== null && !Array.isArray(o)
 const isFunction = f => typeof f === 'function'
 const isPromise = p => isObject(p) && isFunction(p.then)
 
+const handleHandlerError = err => {
+  console.error(err)
+  return false
```

Comment:
What if we replaced

```suggestion
  return Promise.reject(false)
```

and use it also at L26?

Reply:
Because of this: https://github.com/contentful/ui-extensions-sdk/blob/master/lib/app.ts#L63

It expects a handled Promise to come back for it to continue to communicate with the web app. That is why all promises in this function resolve as `false` in the error case.

Reply:
Yeah, sorry. I meant 

```suggestion
  return Promise.resolve(false)
```

As in L26

Reply:
What would be the benefit to doing this? As I see it, I'm already resolving false in my error handlers.

Reply:
It's mostly to have one place where we handle this logic, should we decide to have a custom `onError` handler

Reply:
Oh I see. You're suggesting I replace L26 with this as well and by making the above change I would be able to do that. Got it; can do.